src/python/pants/core/util_rules/adhoc_binaries.py

```diff
@@ -83,22 +82,50 @@ async def download_python_binary(
         ),
     )
 
-    result = await Get(
+    download_result = await Get(
         ProcessResult,
         Process(
             argv=[tar_binary.path, "-xvf", filename],
             input_digest=python_archive,
             env={"PATH": os.pathsep.join(SEARCH_PATHS)},
-            description="Extract Python",
+            description="Extract Pants' execution Python",
```

Comment:
Not important to fix in this PR, but "Pants" is singular in this context, so the possessive is more commonly rendered as "Pants's".